demisto_sdk/commands/lint/linter.py

```diff
@@ -382,8 +382,12 @@ def _gather_facts(self, modules: dict) -> bool:
             if self._facts["docker_engine"]:
                 # Getting python version from docker image - verifying if not valid docker image configured
                 for image in self._facts["images"]:
-                    py_num_version = get_python_version(image=image[0])
-                    py_num = f"{py_num_version.major}.{py_num_version.minor}"
+                    if py_num_version := get_python_version(image=image[0]):
+                        py_num = f"{py_num_version.major}.{py_num_version.minor}"
+                    else:
+                        raise RuntimeError(
```

Comment:
is this the correct exception type to raise in this case?

Reply:
hmm, what do you think the exception should be then?

Reply:
`ValueError`